raster/r.texture/testsuite/test_texture.py

```diff
@@ -23,60 +24,178 @@ def setUpClass(cls):
-        self.assertModule("r.texture", input=self.input, output="corr", method="corr")
-        self.assertRasterFitsUnivar(corr_output, reference=string, precision=2)
-
-    def test_sa(self):
-        """Testing method sa"""
-        sa_SA = "sa_SA"
-        string = """min=95.95
-        max=2942.79
-        cells=1000246"""
-        self.assertModule("r.texture", input=self.input, output="sa", method="sa")
-        self.assertRasterFitsUnivar(sa_SA, reference=string, precision=2)
+        basename = "Corr"
+        method = "corr"
+        output = f"{method}_{basename}"
+        values = """min=-0.533553719520569
+        max=0.443134188652039
+        mean=-0.0511981885382368
+        variance=0.0306650179549719
+        n=996244"""
+        self.assertModule("r.texture", input=self.input, output=method, method=method)
+        self.assertRasterFitsUnivar(output, reference=values, precision=1e-2)
 
     def test_var(self):
         """Testing method var"""
-        var_output = "var_Var"
-        self.assertModule("r.texture", input=self.input, output="var", method="var")
-        self.assertRasterMinMax(
-            map=var_output,
-            refmin=0,
-            refmax=9843.07,
-            msg="var_Var in degrees must be between 0.74 and 9843.07",
-        )
+        basename = "Var"
+        method = "var"
+        output = f"{method}_{basename}"
+        values = """min=0
+        max=8776.787109375
+        mean=90.3218668390769
+        variance=58439.1071892438
+        n=996244"""
+        self.assertModule("r.texture", input=self.input, output=method, method=method)
+        self.assertRasterFitsUnivar(output, reference=values, precision=1e-2)
 
     def test_idm(self):
         """Testing method idm"""
-        idm_IDM = "idm_IDM"
-        self.assertModule("r.texture", input=self.input, output="idm", method="idm")
-        self.assertRasterMinMax(
-            map=idm_IDM,
-            refmin=0,
-            refmax=9843.07,
-            msg="idm_IDM_IDM in degrees must be between 0.74 and 9843.07",
-        )
+        basename = "IDM"
+        method = "idm"
+        output = f"{method}_{basename}"
+        values = """min=0.000490661768708378
+        max=1
+        mean=0.12308521457551
+        variance=0.00497110161251246
+        n=996244"""
+        self.assertModule("r.texture", input=self.input, output=method, method=method)
+        self.assertRasterFitsUnivar(output, reference=values, precision=1e-2)
+
+    def test_sa(self):
+        """Testing method sa"""
+        basename = "SA"
+        method = "sa"
+        output = f"{method}_{basename}"
+        values = """min=95.9583358764648
+        max=2942.79150390625
+        mean=742.367934271396
+        variance=31616.3527726298
+        n=996244"""
+        self.assertModule("r.texture", input=self.input, output=method, method=method)
+        self.assertRasterFitsUnivar(output, reference=values, precision=1e-2)
+
+    def test_sv(self):
```

Comment:
We had two bugs when switching to macOS arm runners (that would occur on M1 and later Macs anyways). 

See https://github.com/OSGeo/grass/issues/3398 and https://github.com/OSGeo/grass/issues/3397, that occurred in the context of this PR https://github.com/OSGeo/grass/pull/3395

Reply:
The test can use _skipif_ decorator with _sys.platfrom.startswith_. There can be multiple tests for different platforms. Ideally, there would be no difference, but that would be out of scope of this PR.

Reply:
Is this deviation caused by `r.texture` or `r.univar`? Either way, it would be good to create an issue for this, so it won't be forgotten. I'm curious whether these deviations are present with non-Apple ARM architectures too. My hunch is that this might be caused by something "trivial" as implicit rounding (via cast), or the like.

Reply:
Thanks for answering. I created an issue #3796 for this.

Reply:
Thanks for the report!